bundles/org.openhab.binding.enocean/README.md

```diff
@@ -223,7 +223,7 @@ If you change the SenderId of your thing, you have to pair again the thing with
 |                                 | enoceanId         | | |
 | environmentalSensor             | receivingEEPId    |                             | A5_13_01 |
 |                                 | enoceanId         | | |
-| centralCommand                  | senderIdOffset    | SenderId used for sending msg.<br/>If omitted, nextSenderId of bridge is used | 1-127 |
+| centralCommand                  | senderIdOffset    | SenderId used for sending msg.If omitted, nextSenderId of bridge is used | 1-127 |
```

Comment:
```suggestion
| centralCommand                  | senderIdOffset    | SenderId used for sending msg. If omitted, nextSenderId of bridge is used | 1-127 |
```

You can also use a double space in Markdown to accomplish the same as a `<br/>` but that is always easily overlooked.